tests/.rubocop.yml

```diff
@@ -8,6 +8,9 @@ Metrics/AbcSize:
 Metrics/CyclomaticComplexity:
   Max: 15
 
+Metrics/LineLength:
```

Comment:
I'd rather disable this on a case-by-case basis (`# rubocop:disable Metrics/LineLength` ... `# rubocop:enable Metrics/LineLength`) rather than changing the line length preference across the whole project.